puppet-lint.gemspec

```diff
@@ -13,9 +13,9 @@ Gem::Specification.new do |spec|
   spec.files = Dir[
     'README.md',
     'LICENSE',
+    '.rubocop.yml',
     'lib/**/*',
     'bin/**/*',
-    'spec/**/*',
```

Comment:
It was already merged, but I'll just leave a note here that this breaks things for plugins. You essentially broke this method:
https://github.com/puppetlabs/puppet-lint/blob/6fc3d5eefc5ba768b7f4c5821c09bd2f26e67c1d/lib/puppet-lint/plugins.rb#L26-L32

Reply:
Oh my bad - must have been on autopilot in vim...

Will restore! Good catch!

Reply:
fixed in #63

Reply:
I'll admit I only know this because in Fedora the spec files are part of `rubygem-puppet-lint-doc` and I had weird failures because I only had `rubygem-puppet-lint` installed. Took me a while to figure out.